src/systemtest/resources/system-test.properties

```diff
@@ -0,0 +1,31 @@
+application.name=eiffel-intelligence
+
+# Test Framework logging level
+logging.level: INFO
+
+# Details for connection to RabbitMQ
+rabbitmq.host: localhost
+rabbitmq.port: 5672
+rabbitmq.user: myuser
+rabbitmq.password: myuser
+
+# Valid TLS versions: 'TLSv1.2', 'TLSv1.1', 'TLSv1', 'TLS', 'SSLv3', 'SSLv2', 'SSL'
+rabbitmq.tlsVersion:
+rabbitmq.exchange.name: ei-exchange
+rabbitmq.domainId: ei-domain
+rabbitmq.componentName:ei-system-test
+rabbitmq.consumerName: messageConsumer
+rabbitmq.queue.durable:true
+rabbitmq.binding.key:
+
+# MongoDB configuration
+mongodb.host: localhost
+mongodb.port: 27017
+mongodb.database: eiffel_intelligence
+mongodb.username:
+mongodb.password:
```

Comment:
Mongodb variables is not needed as well.

Reply:
ok